charmcraft/commands/store/client.py

```diff
@@ -44,11 +45,42 @@
 STORAGE_BASE_URL = 'https://storage.staging.snapcraftcontent.com'
 
 
+def _is_ci_env() -> bool:
+    """Utilize standard env variables to determine if the environment is for testing."""
+    env_prefixes = ["TRAVIS", "AUTOPKGTEST_TMP"]
+    for prefix in env_prefixes:
+        for key in os.environ.keys():
+            if key.startswith(prefix):
+                return True
+    return False
+
+
+def _get_os_platform(filepath: str = "/etc/os-release") -> str:
+    """Determine a system/release combo for an OS using /etc/os-release if available."""
+    if platform.system() == "Linux":
+        os_release = dict()
+        try:
+            with open(filepath) as f:
+                for line in f:
+                    entry = line.rstrip().split("=")
```

Comment:
Just do `key, value = line.strip().split("=", 1)`, no need for subsequent `if` (which will make all this work better if the value has an equal sign).